src/Pages/Shared/ConfigMapSecret/ConfigMapSecretProtected.tsx

```diff
@@ -184,46 +183,47 @@ export const ConfigMapSecretProtected = ({
     )
 
     const renderApproveButton = () => {
-        if (draftData.draftState !== DraftState.AwaitApproval || !ApproveRequestTippy) {
-            return null
+        if (draftData.userApprovalMetadata.hasCurrentUserApproved) {
```

Comment:
TODO: In case of patch strategy we also show an approve button in dry run view pls look into that as well

Reply:
Sure will do after the story is merged